cdc/entry/mounter.go

```diff
@@ -101,7 +101,7 @@ func NewMounter(schemaStorage SchemaStorage,
 // this method could block indefinitely if the DDL puller is lagging.
 func (m *mounterImpl) DecodeEvent(ctx context.Context, pEvent *model.PolymorphicEvent) error {
 	m.metricTotalRows.Inc()
-	if pEvent.RawKV.OpType == model.OpTypeResolved {
+	if pEvent.IsResolved() {
```

Comment:
This is a hot path, and `IsResolved` adds one more check for each event.

Reply:
[Nil-value judgment is removed](https://github.com/pingcap/tiflow/pull/5342/files#diff-15a4777aa4f184040a91f7a61bde28f3d0390fc17709257cec96efe5722bd64bR64), PTAL.